### ISSUE TITLE
Error when running via cron

### ISSUE THREAD

**Issue by Delver26** [lightpohl/gb-dl#8]
Hi, thanks for the awesome tool. I'm loving it so far!
I'm running this on an install of OMV (Debian based) 
I've recently attempted to schedule the script to run as I've been using it to download the most recent 20 videos from the site.

When I run my script manually it works perfectly, but when it runs through cron it throws what looks to be a memory error.
I've attached the output log when running in debug mode. (with API key removed)
[error.log](https://github.com/lightpohl/gb-dl/files/7148585/error.log)

I've also attached the script. Here's the command used for the cron job if that helps.
[gb-dl.js.txt](https://github.com/lightpohl/gb-dl/files/7148584/gb-dl.js.txt)
command:
`cd /configs/giantbombcli/ && /usr/bin/node gb-dl.js`

I'm not familiar with node.js to be honest so I'm not sure if I need to run it in a certain way.
Thanks in advance!

**Comment by lightpohl** [comment]
Hey @Delver26! Thanks for opening an issue — sorry you're running into trouble!

My first thought is something is either going wrong with the archive or network cache files. Would you mind checking a few things for me?

1. What version of Node do you have installed? Should be able to see it with `node --version`.
2. Try running the job with the `--clean` flag added to the `gb-dl` call: it'll skip trying to use the cached network responses. Does that change anything?
3. How large is your archive file getting?

**Comment by Delver26** [comment]
Thanks for the quick response.

1. The Node version is v12.22.6
2. I've added the `--clean` flag to the call but it's still happening.
3. The archive file is only 1.8KB:
![image](https://user-images.githubusercontent.com/5316055/132979542-cd8a2828-04cd-4f43-9618-9e7fcac325a4.png)

**Comment by lightpohl** [comment]
Thank you!

Tested on a couple different devices but I'm not able to reproduce the issue yet. A couple more questions if you don't mind:

1. Does cron run under the same user as you/does it have any memory restrictions?
2. How much memory does this machine have?

Assuming you have greater than 4GB, would you mind updating your script like below and see if it gets any farther?

```bash
NODE_OPTIONS="--max_old_space_size=4096" npx gb-dl --api-key ${apiKey} --debug --video-number ${i} --out-dir ${outDir} --archive --add-guid-prefix
```

**Comment by Delver26** [comment]
Yep no worries!

1. Yeah both running under same user (root)
2. I've got 32GB on the server.

I've tried amending the script as suggested, and I get the below error. 
It does take a lot longer to throw this now, so perhaps the memory limit increase is doing something right 😄  

`/root/.npm/_npx/12373/lib/node_modules/gb-dl/bin/util.js:357
  process.stdout.clearLine();
                 ^

TypeError: process.stdout.clearLine is not a function
    at printProgress (/root/.npm/_npx/12373/lib/node_modules/gb-dl/bin/util.js:357:18)
    at Request.<anonymous> (/root/.npm/_npx/12373/lib/node_modules/gb-dl/bin/util.js:454:11)
    at Request.emit (events.js:314:20)
    at IncomingMessage.<anonymous> (/root/.npm/_npx/12373/lib/node_modules/gb-dl/node_modules/got/dist/source/core/index.js:794:18)
    at Object.onceWrapper (events.js:420:28)
    at IncomingMessage.emit (events.js:326:22)
    at IncomingMessage.origin.emit (/root/.npm/_npx/12373/lib/node_modules/gb-dl/node_modules/@szmarczak/http-timer/dist/source/index.js:43:20)
    at endReadableNT (_stream_readable.js:1241:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
Error: Command failed: NODE_OPTIONS="--max_old_space_size=4096" npx gb-dl --api-key *REMOVED* --debug --video-number 0 --out-dir /media/giantbomb/ --archive --add-guid-prefix
    at checkExecSyncError (child_process.js:635:11)
    at execSync (child_process.js:671:15)
    at Object.<anonymous> (/configs/giantbombcli/gb-dl.js:20:5)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
    at internal/main/run_main_module.js:17:47 {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 12372,
  stdout: null,
  stderr: null
}
Something happened! Moving onto the next video.`

**Comment by lightpohl** [comment]
Ah! `gb-dl` doesn't have some of the non-TTY support I've added to other projects. Let me add that and push up a new version for you to try!

**Comment by lightpohl** [comment]
Okay! Try with the latest version specified and see if you can get a bit further:

```
NODE_OPTIONS="--max_old_space_size=4096" npx gb-dl@2.9.1 --api-key ${apiKey} --debug --video-number ${i} --out-dir ${outDir} --archive --add-guid-prefix
```

**Comment by Delver26** [comment]
That's fixed it, perfect!
Thanks a million 👍🏻

**Comment by lightpohl** [comment]
Awesome! Thanks for the back and forth. 🙏 

Fix with [v2.9.1](https://github.com/lightpohl/gb-dl/commit/52e80f10e62d131de185e22fd09dfb206deaa72e).
